requirements-dev.txt

```diff
@@ -22,13 +24,6 @@ filelock==3.4.0
     # via virtualenv
 identify==2.4.0
     # via pre-commit
-importlib-metadata==5.0.0
```

Comment:
Just curious why this is being removed?

Reply:
requirements-dev.txt is automatically created by running `pip-compile requirements-dev.in` Looks like it's no longer needed by these. Might be because of this: https://realpython.com/lessons/importlibmetadata/